Makefile

```diff
@@ -101,7 +101,17 @@ docker:
 	docker build -t gitea/gitea:latest .
 
 .PHONY: release
-release: release-dirs release-windows release-linux release-darwin release-copy release-check
+release: release-changelog release-dirs release-windows release-linux release-darwin release-copy release-check
+
+.PHONY: release-changelog
+release-changelog: CHANGELOG.md
+
+.PHONY: CHANGELOG.md
+CHANGELOG.md: changelog/unreleased/*.md changelog/HISTORICAL.md
+	cat changelog/HISTORICAL.md >> CHANGELOG.md
```

Comment:
This sounds brutal. A Makefile target that always unconditionally appends to a file ?
I see that file is created by the unreleased/*.md target, but it isn't at all intutitive.